test/e2e-upgrade-kind.sh

```diff
@@ -23,7 +23,6 @@ KOURIER_CONTROL_NAMESPACE=knative-serving
 TEST_NAMESPACE=serving-tests
 
 export KO_DOCKER_REPO=kind.local
```

Comment:
Same with `e2e-kind.sh`. 
Additionally, `KO_DOCKER_REPO` is defined twice in this script L#25 and L#59.